### DISCUSSION TITLE
Add Frame wallet

### DISCUSSION THREAD

**Discussion by JosepBove** [rainbow-me/rainbowkit#357] [Ideas]
Add support for browser-injected frame wallet
https://frame.sh/

I will probably do the PR later

**Comment by nickbytes** [comment]
Cool! Frame is one of the wallets that was on our roadmap to support. 

Let me know when you start working on this, and we can convert to an issue to track there.

Thanks!

**Reply by cdt-eth** [reply]
hey @nickbytes came here to write the same thing.

curious if this ever got worked on?

**Comment by Kharabet** [comment]
@nickbytes any updates on it? Frame wallet became more and more popular. And I see rainbowkit doesn't support it as wallet provider so far

**Comment by nixorokish** [comment]
Hi @nickbytes - here to add my voice to ask for Frame support

**Comment by DanielSinclair** [comment]
This is now available with RainbowKit [1.0.9](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.0.9)
